aiozmq/interface.py

```diff
@@ -194,3 +215,36 @@ def msg_received(self, data):
 
         data is the multipart tuple of bytes with at least one item.
         """
+
+MonitorEvents = {zmq.EVENT_ACCEPTED: 'accepted',
+                 zmq.EVENT_ACCEPT_FAILED: 'accept failed',
+                 zmq.EVENT_BIND_FAILED: 'bind failed',
+                 zmq.EVENT_CLOSE_FAILED: 'close failed',
+                 zmq.EVENT_CLOSED: 'closed',
+                 zmq.EVENT_CONNECTED: 'connected',
+                 zmq.EVENT_CONNECT_DELAYED: 'connect delayed',
+                 zmq.EVENT_CONNECT_RETRIED: 'connect retried',
+                 zmq.EVENT_DISCONNECTED: 'disconnected',
+                 zmq.EVENT_LISTENING: 'listening',
+                 zmq.EVENT_MONITOR_STOPPED: 'monitor stopped'}
+
+
+class ZmqEventProtocol(ZmqProtocol):
```

Comment:
Why do you need new class?
Isn't adding `event_received()` method to `ZmqProtocol` works well?